pkg/apis/core/types.go

```diff
@@ -241,6 +241,11 @@ type PersistentVolumeClaimVolumeSource struct {
 	// will force the ReadOnly setting in VolumeMounts
 	// +optional
 	ReadOnly bool
+	// "NoChange" - Don't change permissions and ownership.  "Always" - Always change the permissions and ownership
+	// to match fsGroup. This is the current behavior and it will be the default one when this proposal is
+	// implemented.  "OnDemand" - Only change permissions when fsGroup of pod and PVC don't match.
+	// +optional
```

Comment:
why is this basically not the same as not specifying fsGroup at all ?
- if fsGroup is not specified then do nothing
- if fsGroup is specified, change ownership of pvc/mounted filessytem
- if supplementalGroups is specified add process to additional groups but dont do anything with filesystem

Reply:
I believe having this field is better because it is explicit, rather than implied behavior.